src/__tests__/select-event.test.tsx

```diff
@@ -146,7 +157,7 @@ describe("The select event helpers", () => {
     );
     expect(form).toHaveFormValues({ food: "" });
 
-    await selectEvent.create(input, "papaya", { createOptionText: "papaya" });
+    await selectEvent.create(input, "papaya", { autoSelect: true, createOptionText: "papaya" });
```

Comment:
Same here - can we remove all the `autoSelect: true`?

Reply:
Ditto.